tests/helics/test_files/encryption_config/openssl_certs/ssl_encryption_config.json.in

```diff
@@ -1,8 +0,0 @@
-{
-    "encrypted": true,
-    "verify_file": "${CMAKE_CURRENT_SOURCE_DIR}/test_files/ca.pem",
```

Comment:
Do we not need this file anymore?

Reply:
It's almost a duplicate of another file one directory up that is used -- I think we use a copy of this file in the GMLC-TDC/networking repository, and it got copied over when I grabbed the certificate and key files to use for CI tests here.